helper.psm1

```diff
@@ -836,9 +833,17 @@ Function launchSingleTests
         {
             Write-Host "Launching $test"
         }
-        If(-Not(Select-String -Path $INNERWORKDIR\ArangoDB\UnitTests\OskarTestSuitesBlackList -pattern $test[0]))
+        If(-Not(Select-String -Path $global:ARANGODIR\UnitTests\OskarTestSuitesBlackList -pattern $test[0]))
         {
-            unittest "$($test[0]) --cluster false --storageEngine $STORAGEENGINE --minPort $global:portBase --maxPort $($global:portBase + 99) $($test[2..$($test.Length)]) --skipNondeterministic true --skipTimeCritical true --testOutput $env:TMP\$($test[0])$($test[1]).out --writeXmlReport false" -output "$INNERWORKDIR\ArangoDB\$($test[0])_$($test[1])"
+            If($test[1])
```

Comment:
May potentially fail if $test[1]=0 in future will contain number as value.

Reply:
In Powershell this test if $test[1] is not empty